docs/source/slack.rst

```diff
@@ -72,6 +72,22 @@ Do not try to assume which pronoun to use while talking to someone. It is much
 nicer if you use gender neutral words (or the names) in any type of
 communication. This guide has more details which one should read.
 
+Adjust notification time for Slack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Slack turns off notifications after 10 PM.
+To avoid missing out on any notifications after that, extend the default
```

Comment:
Can you add a newline between the two paragraphs, please?

Reply:
also, extend the default notification time according to your sleeping schedule.

(Newline)

For changing, reach out to ...

Also, if possible can you attach a screenshot here as well, please?